clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -134,37 +146,110 @@ func (*schedulerLatencyMeasurement) String() string {
 	return schedulerLatencyMetricName
 }
 
-func (s *schedulerLatencyMeasurement) resetSchedulerMetrics(c clientset.Interface, host, provider, masterName string, masterRegistered bool) error {
-	_, err := s.sendRequestToScheduler(c, "DELETE", host, provider, masterName, masterRegistered)
-	if err != nil {
-		return err
+// Helper function to substract two histograms
```

Comment:
the golang convention is to start comment with name of the field/function/...

// histogramSub is a helper function to substract two histograms

Reply:
fixed